lib/releaselib.sh

```diff
@@ -1025,25 +1027,25 @@ release::docker::release () {
   local version=$2
   local build_output=$3
   local release_images=$build_output/release-images
-  local docker_target
   local arch
   local -a arches
   local tarfile
   local orig_tag
-  local -a new_tags
   local new_tag
   local binary
   local -A manifest_images
 
+  # TODO(vdf): Remove all GCRIO_PATH_PROD_PUSH logic once the k8s.gcr.io vanity
+  #            domain flip (VDF) is successful
   if [[ "$registry" == "$GCRIO_PATH_PROD" ]]; then
     # Switch to the push alias if using the $GCRIO_PATH_PROD alias
     push_registry="$GCRIO_PATH_PROD_PUSH"
   fi
 
   logecho "Send docker containers from release-images to $push_registry..."
 
-  arches=($(cd "$release_images"; echo *))
-  for arch in ${arches[@]}; do
+  arches=($(ls "$release_images"))
+  for arch in "${arches[@]}"; do
```

Comment:
you can do:
```bash
for arch in "${release_images}/*"; do
```

Reply:
I went with:

```shell
  mapfile -t arches < <(find "${release_images}" -maxdepth 1 -mindepth 1 -type d -exec basename {} \;)
  for arch in "${arches[@]}"; do
```

based on the guidance in shellcheck SC2207: https://github.com/koalaman/shellcheck/wiki/SC2207

Using `find` and `basename` takes some of the unexpected `*` behavior out of the equation.